diff -Nur libspectre-0.2.0/configure libspectre-0.2.0.new/configure
--- libspectre-0.2.0/configure	2008-01-03 11:18:06.000000000 -0500
+++ libspectre-0.2.0.new/configure	2008-01-03 16:05:55.753081110 -0500
@@ -21886,7 +21886,7 @@
 
 
 
-ac_config_files="$ac_config_files Makefile Doxyfile libspectre/Makefile libspectre/spectre-version.h libspectre/ghostscript/Makefile test/Makefile libspectre.pc"
+ac_config_files="$ac_config_files Makefile Doxyfile libspectre/Makefile libspectre/spectre-version.h test/Makefile libspectre.pc"
 
 cat >confcache <<\_ACEOF
 # This file is a shell script that caches the results of configure
@@ -22485,7 +22485,6 @@
     "Doxyfile") CONFIG_FILES="$CONFIG_FILES Doxyfile" ;;
     "libspectre/Makefile") CONFIG_FILES="$CONFIG_FILES libspectre/Makefile" ;;
     "libspectre/spectre-version.h") CONFIG_FILES="$CONFIG_FILES libspectre/spectre-version.h" ;;
-    "libspectre/ghostscript/Makefile") CONFIG_FILES="$CONFIG_FILES libspectre/ghostscript/Makefile" ;;
     "test/Makefile") CONFIG_FILES="$CONFIG_FILES test/Makefile" ;;
     "libspectre.pc") CONFIG_FILES="$CONFIG_FILES libspectre.pc" ;;
 
diff -Nur libspectre-0.2.0/configure.ac libspectre-0.2.0.new/configure.ac
--- libspectre-0.2.0/configure.ac	2008-01-03 11:12:50.000000000 -0500
+++ libspectre-0.2.0.new/configure.ac	2008-01-03 16:05:55.753081110 -0500
@@ -199,7 +199,6 @@
 Doxyfile
 libspectre/Makefile
 libspectre/spectre-version.h
-libspectre/ghostscript/Makefile
 test/Makefile
 libspectre.pc
 ])
diff -Nur libspectre-0.2.0/libspectre/Makefile.am libspectre-0.2.0.new/libspectre/Makefile.am
--- libspectre-0.2.0/libspectre/Makefile.am	2007-12-09 12:18:09.000000000 -0500
+++ libspectre-0.2.0.new/libspectre/Makefile.am	2008-01-03 16:06:20.542081871 -0500
@@ -1,4 +1,3 @@
-SUBDIRS = ghostscript
 
 INCLUDES = 			\
 	-I$(top_builddir)	\
diff -Nur libspectre-0.2.0/libspectre/Makefile.in libspectre-0.2.0.new/libspectre/Makefile.in
--- libspectre-0.2.0/libspectre/Makefile.in	2008-01-03 11:18:05.000000000 -0500
+++ libspectre-0.2.0.new/libspectre/Makefile.in	2008-01-03 16:06:40.182081456 -0500
@@ -210,7 +210,6 @@
 sharedstatedir = @sharedstatedir@
 sysconfdir = @sysconfdir@
 target_alias = @target_alias@
-SUBDIRS = ghostscript
 INCLUDES = \
 	-I$(top_builddir)	\
 	-I$(srcdir)		\
